src/openrct2-ui/windows/Sign.cpp

```diff
@@ -127,7 +127,7 @@ class SignWindow final : public Window
 
         // Create viewport
         rct_widget& viewportWidget = window_sign_widgets[WIDX_VIEWPORT];
-        const auto focus = Focus2(CoordsXYZ{ signViewPosition, viewZ });
+        const auto focus = Focus(CoordsXYZ{ signViewPosition, viewZ });
```

Comment:
This one and other locals named `focus` need updating, as they now hide the class member with the same name.